content/blog/_posts/2021-04-rusty-workflows.md

```diff
@@ -0,0 +1,601 @@
+---
+title: Improving our Rusty GitHub workflows
```

Comment:
This title is a good start. I think it should include "Github Actions" and "Rust" for the SEO. But what would you have searched for if you were looking to find this to solve the same problem?